docs/Examples.md

```diff
@@ -7,6 +7,7 @@ This creates a default Graph client that uses `https://graph.microsoft.com` as t
 
 use Microsoft\Graph\GraphRequestAdapter;
 use Microsoft\Graph\GraphServiceClient;
+use Microsoft\Kiota\Authentication\Oauth\AuthorizationCodeContext;
 use Microsoft\Kiota\Authentication\Oauth\ClientCredentialContext;
```

Comment:
I think having both examples in the same code block may be confusing. I think it would be clearer to separate the 2 examples and provide recommendations for when to use each.

Reply:
Agreed. Will update

Reply:
Added more info on the section